src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,176 @@
 package core.basesyntax;
 
+import core.basesyntax.service.ObjectService;
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        Node<T> node = new Node<>(tail, value, null);
+        if (head == null) {
+            head = node;
+        }
+        if (tail != null) {
+            tail.next = node;
+        }
+        tail = node;
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
+        if ((index == 0 && size == 0) || size == index) {
+            add(value);
+            return;
+        }
+        checkIndexOfBoundsException(index);
+        Node<T> foundedNode = getNodeByIndex(index);
+        if (foundedNode != null) {
+            insertNewNode(value, foundedNode);
+        } else {
+            createNewNodeAtTail(value);
```

Comment:
you can use method add()
```suggestion
            add(value);
```